sqlalchemy_ibmi/base.py

```diff
@@ -636,19 +637,41 @@ def get_rowcount(self):
     def fire_sequence(self, seq, type_):
         return self._execute_scalar(
             "SELECT NEXTVAL FOR " +
-            self.connection.dialect.identifier_preparer.format_sequence(seq) +
+            self.connection.dialect.preparer.format_sequence(seq) +
```

Comment:
I think this should be left as identifier_preparer.

dialect here should be our dialect, which subclasses DefaultDialect which sets up `identifier_preparer` here: https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/engine/default.py#L249, while its `preparer` is actually a class not an instance: https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/engine/default.py#L48 and we have the same in our dialect: https://github.com/IBM/sqlalchemy-ibmi/blob/master/sqlalchemy_ibmi/base.py#L677

I can understand the confusion because in the SQLCompiler class, which Db2Compiler subclasses, it sets `preparer` to be the dialect's `identifier_preparer`: https://github.com/sqlalchemy/sqlalchemy/blob/master/lib/sqlalchemy/sql/compiler.py#L411